packages/anon-creds/src/models/exchange.ts

```diff
@@ -0,0 +1,104 @@
+// TODO: Maybe we can make this a bit more specific?
+export type WalletQuery = Record<string, unknown>
+
+export interface NonRevokedInterval {
+  from?: number | undefined
+  to?: number | undefined
+}
+
+export interface AnonCredsCredentialOffer {
+  schema_id: string
+  cred_def_id: string
+  nonce: string
+  key_correctness_proof: Record<string, unknown>
```

Comment:
Would it be possible, or maybe later, to make this more explicit? If we can type the fields in here it is always a nice benefit. We can also pick this up later and just create and issue now if we want this.

Reply:
This comment basically applies to any `Record<string, unknown>`